frontend/wallet/src/common/Bindings.re

```diff
@@ -202,7 +203,11 @@ module LocalStorage = {
   external getItem:
     (
     [@bs.string]
-    [ | [@bs.as "network"] `Network | [@bs.as "addressbook"] `AddressBook]
+    [
+      | [@bs.as "network"] `Network
+      | [@bs.as "addressbook"] `AddressBook
+      | [@bs.as "onboarding"] `Onboarding
+    ]
```

Comment:
Looks like some onboarding stuff snuck in

Reply:
Whoops I'll take this out.